### PR TITLE
Updated default.conf & README

### PR DESCRIPTION
**Pull request by JHaydock-Pro**
Updated default.conf & README to include new lines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,37 +12,10 @@ python-novaclient | 3.3.1
...
-cloudPlatform : openstack or opennebula
-aqProfiles: URL of aquilon profiles directory
-```
-
-
 Launching
 ===
 
-To launch the webfrontend without Apache, create `config/global.conf` and a `badwords` file using the guide above then run the following:
+To launch the webfrontend without Apache, create `config/global.conf` and a `badwords` file using the guide below then run the following.
```
Comment (tofu-rocketry) [inline README.md:18]: Why the change to a full stop?

File: README.md [hunk 2]
```diff
@@ -54,5 +27,78 @@ When running the webfrontend without Apache, you might have to manually launch w
...
+auth.ldap_address = The URL of an LDAP server to convert FedID into real name (optional)
+auth.ldap_basedn = The basedn to lookup in LDAP (optional)
+auth.session_expire_secs = Number of seconds to wait to expire a session
+
+# CherryPy Server
+server.socket_host = Network interface CherryPy will listen to
+server.socket_port = TCP port CherryPy will listen to
+
+# Cherrypy Tools
+tools.caching.on = False
```
Comment (tofu-rocketry) [inline README.md]: What do these `False` and `True` options actually do?

File: README.md [hunk 3]
```diff
@@ -54,5 +27,78 @@ When running the webfrontend without Apache, you might have to manually launch w
...
+tools.isAuthorised.on = False
+tools.jinja.on = True
+tools.sessions.on = True
+tools.sessions.storage_type = 'File'
```
Comment (tofu-rocketry) [inline README.md]: What's this do? What options can you pick from?

File: README.md [hunk 4]
```diff
@@ -54,5 +27,78 @@ When running the webfrontend without Apache, you might have to manually launch w
...
+openstack_default_domain = Domain for user logins
+vmNetworkLabel = Network label to assign to VMs
+securityGroupName = Security group to assign to created VMs
+flavorPrefix = Prefix to available flavor names
+availabilityZoneName = Availabilty zone to assign to VMs
+aqProfiles = URL storing aquilon profiles
+countLimit = Maximum number of VMs created in OpenStack at once (If quota is unlimited)
+
+# Misc
+response.headers.server = "Nothing to see here ... move along :)"
```
Comment (tofu-rocketry) [inline README.md]: ???

File: config/default.conf [hunk 5]
```diff
@@ -1,42 +1,63 @@
...
-wsport: 29876
-#wscert: "/etc/httpd/ssl/CERT"
-#wskey: "/etc/httpd/ssl/KEY"
-wstokendir: "/var/www/frontend/tokens/"
-
-# set to True if using Apache
-wsgi_enabled: True
+# Auth
+auth.ldap_address = "ldap://LDAP"
+auth.ldap_basedn = "dc=FED,dc=CCLRC,dc=AC,dc=UK"
```
Comment (tofu-rocketry) [inline config/default.conf]: Is this correct? Shouldn't the keys be different and capitalised?

Also, should it not be a generic version in this repository?

File: config/default.conf [hunk 6]
```diff
@@ -1,42 +1,63 @@
...
+openstack_default_domain = "USER_DOMAIN"
+vmNetworkLabel = "NETWORK_LABEL"
+securityGroupName = "SEC_GROUP"
+flavorPrefix = "m"
+availabilityZoneName = "AVAILIABILTY_ZONE"
+aqProfiles = "AQUILON_URL/profiles/"
+countLimit = 10
+
+# Misc
+response.headers.server = "Nothing to see here ... move along :)"
```
Comment (tofu-rocketry) [inline config/default.conf:62]: ???

Comment (tofu-rocketry) [reply config/default.conf:62]: Are you going to replace this with something similar to the readme?

Comment (JHaydock-Pro) [reply config/default.conf:62]: No, that string is the actual value for this key, set by Callum Williams. I have no reason to believe it should be changed.

Comment (tofu-rocketry) [reply config/default.conf:62]: Hmm, ok.

**Review by tofu-rocketry — APPROVED** [review]

**Review by jrha — APPROVED** [review]
LGTM
We can always put the full-stop back to a colon...
